packages/traceability-schemas/schemas/BillOfLading.json

```diff
@@ -20,20 +20,62 @@
       ]
     },
     "billOfLadingNumber": {
-      "$comment": "{\"term\": \"billOfLadingNumber\", \"@id\": \"https://schema.org/identifier\"}",
+      "$comment": "{\"term\": \"billOfLadingNumber\", \"@id\": \"https://service.unece.org/trade/uncefact/vocabulary/uncl1153/#Bill_of_lading_number\"}",
```

Comment:
Not sure why this is added?

this appears to be a Buy Ship Pay Reference Data Model  vocabulary, not entirely sure this is appropriate for the generic BillOfLading

Reply:
@mkhraisha, sticking with schema.org/identifier. I might raise a separate PR/issue sometime dedicated to continue the discussion.